### ISSUE TITLE
Icons cannot be aria-describedby

### ISSUE THREAD

**Issue by joshhunt** [grafana/grafana#76762]
Identified in https://github.com/grafana/grafana/pull/76683, our current Icon component does not work with aria-describedby [because](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby)

> The aria-describedby attribute can be used with semantic HTML elements and with elements that have an ARIA [role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles).

And our Icon wraps the SVG with a div that prevents aria-describeddb from working properly.

To fix this we should:
 - evaluate whether the wrapping div is required in Icon
 - ensure that the root element of the Icon component has a role that can recieve aria-labelledby/aria-describeddb

**Comment by DerenToy** [comment]
Hello,

I'm new here, and I'm really excited to start contributing for the first time, and I really want to do this.

However, since I'm new, I'd like to ask a few questions, if you don't mind. Do you need to assign the task to me for me to work on, or should I fork the project myself, complete the task, and then open a pull request to notify you? Also, is there a specific timeframe within which I should complete the task?

Thank you very much in advance.
